src/seedu/addressbook/data/person/Address.java

```diff
@@ -40,6 +41,11 @@ public static boolean isValidAddress(String test) {
     public String toString() {
         return value;
     }
+   
+    @Override
+    public String getPrintableString(Printable... printables){
```

Comment:
May have misunderstood the question :-).

The activity has 2 parts:
a. New interface, with a getPrintableString()  (no parameter), with suitable class implementations.
b. A new method getPrintableString(Printable... )